powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/service/CredentialService.java

```diff
@@ -73,9 +75,10 @@
      * @param credentialValidationService Credential validation service.
      * @param credentialProtectionService Credential protection service.
      * @param endToEndEncryptionService End-to-end encryption service.
+     * @param userIdentityRepository User identity repository.
      */
     @Autowired
-    public CredentialService(UserIdentityLookupService userIdentityLookupService, CredentialDefinitionService credentialDefinitionService, CredentialRepository credentialRepository, CredentialHistoryService credentialHistoryService, IdGeneratorService idGeneratorService, NextStepServerConfiguration nextStepServerConfiguration, CredentialGenerationService credentialGenerationService, CredentialValidationService credentialValidationService, CredentialProtectionService credentialProtectionService, EndToEndEncryptionService endToEndEncryptionService) {
+    public CredentialService(UserIdentityLookupService userIdentityLookupService, CredentialDefinitionService credentialDefinitionService, CredentialRepository credentialRepository, CredentialHistoryService credentialHistoryService, IdGeneratorService idGeneratorService, NextStepServerConfiguration nextStepServerConfiguration, CredentialGenerationService credentialGenerationService, CredentialValidationService credentialValidationService, CredentialProtectionService credentialProtectionService, EndToEndEncryptionService endToEndEncryptionService, UserIdentityRepository userIdentityRepository) {
```

Comment:
This is a candidate for some service catalogue. We are autowiring the universe here. We can leave it for now but let's have a look at this later. (I will open an issue.)

Reply:
Yes, I am aware of this issue.